### DISCUSSION TITLE
Debian 12 support?

### DISCUSSION THREAD

**Discussion by DaniilDBorisov** [ComplianceAsCode/content#11218] [Q&A]
Basically the title. My management wants us to use oscap in conjunction with SSGs provided here to monitor our system vulnerabilities except our machines are based on Debian 12. Is there any news on adding Debian 12 SSG to the list?

**Comment by Mab879** [comment]
Thanks for the question.

Content for each distro is added by the organization behind it (e.g. Red Hat and RHEL, Canonical and Ubuntu) or by the community. So far no one has contributed content for Debian 12 but we be happy to accept contributions. 

Please see the [docs](https://complianceascode.readthedocs.io/en/latest/manual/developer/03_creating_content.html#creating-a-new-product) for some guidance. Also we would happy to answer questions here as well.
